pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -52,5 +52,5 @@ func (p *TaintToleration) Filter(ctx context.Context, placement *policyv1alpha1.
 		return framework.NewResult(framework.Success)
 	}
 
-	return framework.NewResult(framework.Unschedulable, fmt.Sprintf("cluster(s) had untolerated taint {%s: %s}", taint.Key, taint.Value))
+	return framework.NewResult(framework.Unschedulable, fmt.Sprintf("cluster(s) had untolerated taint {key: %s, value: %s, effect: %s}", taint.Key, taint.Value, taint.Effect))
```

Comment:
What about `taint.ToString()`? I'm afraid taint value could be empty here.

Reply:
ok